app/controllers/chatrooms_controller.rb

```diff
@@ -1,2 +1,9 @@
 class ChatroomsController < ApplicationController
+  def index
+    @chatrooms = Chatroom.all
```

Comment:
I don't think this would be all of the chatrooms, otherwise everyone can see every chatroom that's been created. It should only be the chatroom where the user is involved. Something like `@chatrooms = Chatroom.where something` maybe? I'd have to rewatch the lecture to confirm.